### PR TITLE
fix(backup): Add option model query export test

### PR DESCRIPTION
**Pull request by azaslavsky**
This test is a follow on to the quick fix deployed in https://github.com/getsentry/sentry/pull/68323.

### CODE REVIEW

File: src/sentry/models/options/option.py [hunk 1]
```diff
@@ -54,9 +54,9 @@ def query_for_relocation_export(cls, q: models.Q, pk_map: PrimaryKeyMap) -> mode
         return q & ~models.Q(
             key__in={
                 "sentry:install-id",  # Only used on self-hosted
+                "sentry:latest_version",  # Auto-generated periodically, which defeats comparison
```
Comment (maxwelllwang) [inline src/sentry/models/options/option.py:57]: why the change in order? does it have any difference or is it just formatting alphabetical thing?

Comment (azaslavsky) [reply src/sentry/models/options/option.py:57]: No difference, just realized I alphabetized incorrectly when I did in the previously mentioned PR.

**Review by maxwelllwang — APPROVED** [review]
tests
